how-to/customize-workspace/client/src/framework/workspace/notifications.ts

```diff
@@ -30,13 +30,15 @@ export async function register(): Promise<RegistrationMetaInfo> {
 			const settings = await getSettings();
 			const notificationPlatformSettings = settings?.notificationProvider;
 			if (notificationPlatformSettings !== undefined) {
-				try {
-					await registerPlatform(notificationPlatformSettings);
-					notificationsRegistered = true;
-					logger.info("Registered notifications");
-				} catch (error) {
-					logger.error("We were unable to register with Notification Center", error);
-				}
+				registerPlatform(notificationPlatformSettings)
+					.then(() => {
+						notificationsRegistered = true;
+						logger.info("Registered notifications");
+						return true;
```

Comment:
No need to return true, it doesnt get used

Reply:
This is the linting rules that were specified. That something must always be returned (if you just return it then complains about a pointless return).